labs/b5.md

```diff
@@ -0,0 +1,371 @@
+    
+6. `ip`
+`ip` is a command with many subcommands offering a lot of functionality -- so much it can be overwhelming at first. You will most commonly be using ip to display/modify routing, IP addresses, or network interfaces.
+
+    It will take time to get use to how much functionality is included in this command but for reference here is a pretty compact [cheatsheet]. A few common use cases include: `ip addr` which displays information on your IP addresses, `ip route` which displays information on your routing table, and `ip link` which displays information about your network interfaces.
+
+[cheatsheet]: https://access.redhat.com/sites/default/files/attachments/rh_ip_command_cheatsheet_1214_jcs_print.pdf
+
+7. `netstat`
+This tool is good for printing network connections, routing tables, and probing sockets, amongst other functions.
+
+    `netstat` also has functionality to probe sockets for activity and displays information such protocol (UDP/TCP)
+    
+    If you are investigating sockets `ss` and `lsof` are also options you may want to consider
+8. `tcpdump`
+Perfect for monitoring incoming or outgoing traffic on a machine. 
+
+    `tcpdump` offers countless options when it comes to analyzing traffic: it can capture packets, log traffic, compute metrics, filter traffic, monitor specific interfaces, etc. As a primer you can check out these [examples][tcp].
+
+[tcp]: https://www.rationallyparanoid.com/articles/tcpdump.html
+
+9. `nc`
+Netcat is a very powerful tool that can be used for just about anything involving TCP or UDP.  It can open TCP connections, send UDP packets, listen on arbitrary TCP and UDP ports, do port scanning, and deal with both IPv4 and IPv6.
+
+10. `curl`
+cURL does as its name suggests, and allows you to see the contents at certain URLs. Beyond this it's also an extremely powerful program that lets you interact with and inspect servers over several different protocols certain protocols such as HTTP, FTP, etc ... 
+
+    Be sure to check out its [documentation][curl] for specific use cases.
+
+[curl]: https://curl.haxx.se/docs/
+
+11. `wget`
+`wget` is quite similar to `curl` in the sense that they are both command line tools designed to transfer data from or to servers with certain protocols and both come with a bunch of features.
+
+    There are differences between the commands, two notable examples being that wget is command line only meaning there no library or API.  However, wget has a big advantage of being able to download recursively. You can read a bit more on the two tools [here][curl vs wget].
+
+[curl vs wget]: https://daniel.haxx.se/docs/curl-vs-wget.html
+
+
+# Exercises
+
+[checkoff]: https://goo.gl/forms/j17WWRXOIvwabPUl2
+[files]: https://github.com/0xcf/decal-labs
+
+**Please complete this lab on your student VMs**
+
+**Additionally, These questions can also be found on the lab checkoff form located [here][checkoff] so feel free to read them there instead.**
```

Comment:
NIT: `These` -> `these`

Reply:
Also, I think the checkoff form link should be replicated at the end, since that's where the other labs have mostly had the link so students will probably look there first and be confused or something. Not a big deal either way, but I think it could help clarify some confusion.